test/Azure.Functions.Cli.Tests/E2E/Helpers/RunConfiguration.cs

```diff
@@ -17,8 +18,10 @@ public class RunConfiguration
         public string[] OutputDoesntContain { get; set; } = Array.Empty<string>();
         public string[] ErrorDoesntContain { get; set; } = Array.Empty<string>();
         public Action<string> PreTest { get; set; }
-        public Func<string, Process, Task> Test { get; set; }
+        public Func<string, Process, StringBuilder, Task> Test { get; set; }
         public TimeSpan CommandTimeout { get; set; } = TimeSpan.FromSeconds(40);
         public string CommandsStr => $"{string.Join(", ", Commands)}";
+        public bool WaitForRunningHostState = false;
+        public string HostProcessPort = "7071";
```

Comment:
is 7071 going to be the default when running the E2E tests?

Reply:
7071 is the default port when you run func start, unless you specify "--port" in the start command. I believe 90% of our tests use the default and the remaining 10ish % provide a different port